desktop/src/main/java/bisq/desktop/util/GUIUtil.java

```diff
@@ -268,6 +275,21 @@ public static void importAccounts(User user, String fileName, Preferences prefer
         }
     }
 
+    public static void exportJSON(String fileName, JsonElement data, Stage stage) {
```

Comment:
Is there a pretty print feature in the json framework? Would be nice to have, but if not no worry.